### PR TITLE
[Platform.sh] Further platform.sh improvements

### PR DESCRIPTION
**Pull request by lchrusciel**

### CODE REVIEW

File: .platform.app.yaml [hunk 1]
```diff
@@ -9,6 +9,11 @@ relationships:
     database: "mysql:mysql"
     redis: "redis:redis"
 
+
```
Comment (pamil) [inline .platform.app.yaml]: Doubled new line

File: .platform.app.yaml [hunk 2]
```diff
@@ -9,6 +9,11 @@ relationships:
...
+variables:
+    env:
+        SYMFONY_ENV: 'prod'
```
Comment (pamil) [inline .platform.app.yaml]: Are you sure it's `SYMFONY_ENV`, not `APP_ENV`?

Comment (lchrusciel) [reply .platform.app.yaml]: Yup Ref: https://github.com/Sylius/SyliusDemo/blob/master/bin/console#L20

**Comment by pamil** [comment]
Thank you! 🎉

**Pull request merged**